src/cascadia/TerminalApp/CommandPalette.xaml

```diff
@@ -153,12 +176,21 @@ the MIT License. See LICENSE in the project root for license information. -->
                                        HorizontalAlignment="Left"
                                        Text="{x:Bind Name, Mode=OneWay}" />
 
-                            <TextBlock Grid.Column="2"
-                                       Foreground="{ThemeResource SystemControlForegroundBaseMediumBrush}"
-                                       FontSize="12"
-                                       HorizontalAlignment="Right"
-                                       VerticalAlignment="Center"
-                                       Text="{x:Bind KeyChordText, Mode=OneWay}" />
+
+                            <Grid Grid.Column="2"
```

Comment:
You're pretty much using a `Grid` in place of a `Border`, right? ... can we just use `Border`?

Reply:
Uh probably, this was just the first thing that I got to work right.